vhost-user-backend/Cargo.toml

```diff
@@ -19,12 +19,12 @@ log = "0.4.17"
 userfaultfd = { version = "0.8.1", optional = true }
 vhost = { path = "../vhost", version = "0.11", features = ["vhost-user-backend"] }
 virtio-bindings = "0.2.1"
-virtio-queue = "0.12.0"
-vm-memory = { version = "0.14.0", features = ["backend-mmap", "backend-atomic"] }
```

Comment:
Why we need to enable `backend-bitmap` with this update?

Reply:
Sorry, it is fixed now.